packages/synapse-react-client/src/components/UserCard/UserCardMedium.tsx

```diff
@@ -251,7 +243,8 @@ export function UserCardMedium({
   return (
     <Card>
       <div
-        className={`SRC-primary-background-color SRC-userCard SRC-userCardMediumUp`}
+        className={`SRC-userCard`}
```

Comment:
Why remove the `SRC-userCardMediumUp` style? When I look at this in Storybook, it looks like that caused us to lose some crucial padding in the top + left part of the card

![image](https://github.com/user-attachments/assets/48b6c2db-dbdb-43ab-8395-fe5c9b0bdbce)

Reply:
I added this back - I agree I don't think it needed to be removed, thanks!